src/client/pages/OfferNew/Introduction/Sidebar/DiscountCodeModal.tsx

```diff
@@ -132,6 +128,10 @@ export const DiscountCodeModal: React.FC<Props> = ({
   const textKeys = useTextKeys()
   const [redeemCode] = useRedeemCodeMutation()
 
+  const discountSchema = Yup.object({
+    code: Yup.string().required(textKeys.SIDEBAR_ADD_DISCOUNT_ERROR()),
```

Comment:
same here, i'm slightly sure we run it through translations twice now :) https://github.com/HedvigInsurance/web-onboarding/blob/50413cc7edff5c71383ae95d99f802589dd15cfa/src/client/pages/OfferNew/Introduction/Sidebar/DiscountCodeModal.tsx#L211

Reply:
YES you're right - see your comment above for answer! 🧠